src/features/surveys/components/surveyForm/SurveySignature.tsx

```diff
@@ -0,0 +1,143 @@
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import useCurrentUser from 'features/user/hooks/useCurrentUser';
+import {
+  Box,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { FC, useCallback, useState } from 'react';
+import {
+  ZetkinSurveyExtended,
+  ZetkinSurveySignatureType,
+} from 'utils/types/zetkin';
+
+export type SurveySignatureProps = {
+  formData: NodeJS.Dict<string | string[]>;
+  survey: ZetkinSurveyExtended;
+};
+
+const SurveySignature: FC<SurveySignatureProps> = ({ formData, survey }) => {
+  const currentUser = useCurrentUser();
+  const theme = useTheme();
+
+  const [signatureType, setSignatureType] = useState<
+    ZetkinSurveySignatureType | undefined
+  >(formData['sig'] as ZetkinSurveySignatureType | undefined);
+
+  const handleRadioChange = useCallback(
+    (value: ZetkinSurveySignatureType) => {
+      setSignatureType(value);
+    },
+    [setSignatureType]
+  );
+
+  const selectedStyle = {
+    backgroundColor: '#fbcbd8',
```

Comment:
@richardolsson I've not been able to track down this color or a similar one in the theme to satisfy [your comment](https://github.com/zetkin/app.zetkin.org/pull/1636/files#r1395153304). Want me to add it? Or is this just my lack of familiarity with the theme?

Reply:
Let's leave it as is for now until we get some designers to look at this before launching it. Thanks for pointing it out!